etc/templates/web/nginx-location.conf.template

```diff
@@ -0,0 +1,30 @@
+#sendfile on;
```

Comment:
OK, I get you. Just worried that it's another piece of commented duplicated code that will never be tested. It's essentially `omero web config nginx` with the `location` blocks replaced by a single `include /opt/omero/web/omero-web-location.include;` so if we do modify the existing `nginx` templates we should also modify this comment.